app/src/main/java/com/github/se/polyfit/data/local/database/MealDatabase.kt

```diff
@@ -13,14 +16,26 @@ import com.google.common.reflect.TypeToken
 import com.google.gson.Gson
 import java.time.LocalDate
 
-@Database(entities = [MealEntity::class], version = 1)
+@Database(
+    entities = [MealEntity::class],
+    version = 3,
+    autoMigrations =
+        [
+            AutoMigration(from = 1, to = 2, spec = MealDatabase.RemoveMealId::class),
+            AutoMigration(from = 2, to = 3, spec = MealDatabase.RemoveFirebaseId::class)])
```

Comment:
We could, but this just makes it easier for all developers. No one has to delete and restart their DB, instead when they run the code it will automatically migrate the database

Reply:
And yeah i think its just so that everytime the database is created it builds properly. Im actually not so sure if we need it after or not, but it definitely doesn't hurt anything afaik)

Reply:
That's awesome! I tested the app by creating a bunch of meals using the older format and then updated the app. The migration worked fine with no crashes or weird behaviors.